packages/apidom-ns-workflows-1/src/refractor/visitors/workflows-1/ParametersVisitor.ts

```diff
@@ -0,0 +1,36 @@
+import stampit from 'stampit';
```

Comment:
Should be located in `workflows-1/step/ParametersVisitor.ts`

Reply:
Thanks - nice spot. Location changed